src/systemdhook.c

```diff
@@ -1089,7 +1090,7 @@ int main(int argc, char *argv[])
 
 		pr_pdebug("%s: UID: %d", id, uid);
 
-		if (prestart(rootfs, id, target_pid, mount_label, config_mounts, config_mounts_len, uid, gid) != 0) {
+		if (prestart(rootfs, container_uuid, id, target_pid, mount_label, config_mounts, config_mounts_len, uid, gid) != 0) {
```

Comment:
do you want the id variable in here?  I thought container_uuid would replace it, not be added to the list of params.

Reply:
This looks like container_uuid is the string "container_uuid=UUID"

Reply:
The variable id comes from the shortid function,  which seems to only be used for logging purposes at the moment (it's 12 chars long).